Stanford/JumpstartEngineering/Install/Users/UserOne.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @file
+ * Update email address for user 1.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\Users;
+/**
+ *
+ */
+class UserOne extends \AbstractInstallTask {
```

Comment:
This function should probably replace the JSA one where it creates the administrator rather than updating it.